quantum/process_keycode/process_unicode_common.h

```diff
@@ -23,8 +23,13 @@
 #define UNICODE_TYPE_DELAY 10
 #endif
 
-__attribute__ ((unused))
-static uint8_t input_mode;
+typedef union {
+  uint32_t raw;
+  struct {
+    uint8_t  input_mode      :8;
```

Comment:
```suggestion
    uint8_t input_mode :8;
```
This spacing seems a bit much for now.